tensorflow_addons/image/filters_test.py

```diff
@@ -94,10 +94,15 @@ def test_invalid_filter_shape(self):
         msg = ("The `filter_shape` argument must be a tuple of 2 integers.")
         image = tf.ones(shape=(1, 28, 28, 1))
 
-        for filter_shape in [(3, 3, 3), (3, None, 3), None]:
+        for filter_shape in [(3, 3, 3), (3, None, 3)]:
             with self.subTest(filter_shape=filter_shape):
                 with self.assertRaisesRegexp(ValueError, msg):
                     mean_filter2d(image, filter_shape=filter_shape)
+        
+        for filter_shape in [None]:  
+            with self.subTest(filter_shape=filter_shape):
+                with self.assertRaisesRegexp(TypeError, msg):
```

Comment:
Does it make sense? cc  @WindQAQ

Reply:
It looks weird, but I have to separate `None` and other tests. May I have your suggestions?

Reply:
Probably get rid of the first for loop?

Reply:
I think that's why core TF raises `ValueError` though

Reply:
I am not sure if it makes sense with the way I modified.